templates/core/layout/region--filters.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+/**
+ * @file
+ * Theme override to display a region.
+ *
+ * Available variables:
+ * - content: The content for this region, typically blocks.
+ * - attributes: HTML attributes for the region <div>.
+ * - region: The name of the region variable as defined in the theme's
+ *   .info.yml file.
+ *
+ * @see template_preprocess_region()
+ */
+#}
+{%
+  set classes = [
+  'filter-section',
+  'sidebar',
+  'modal',
+]
+%}
+{% if content %}
+  <aside id="filter" {{ attributes.addClass(classes) }} tabindex="-1" aria-hidden="false">
```

Comment:
Is not an aside, it relates directly to the content and on mobile it's technically a 'dialog' (but let's not go down that road just yet).